compiler/modules/print.wasm/binaryen.bal

```diff
@@ -10,60 +10,127 @@ type Export record {
 };
 
 type Expression record {
```

Comment:
If `Expression` is just here as a super type (without it's own attributes) you may do `type Expression IfExpr|GetLocalExpr|...`.